single-line-diagram-iidm-extensions/src/main/java/com/powsybl/sld/iidm/extensions/BusbarSectionPositionAdderImpl.java

```diff
@@ -0,0 +1,34 @@
+package com.powsybl.sld.iidm.extensions;
+
+import com.powsybl.commons.extensions.AbstractExtensionAdder;
+import com.powsybl.iidm.network.BusbarSection;
+
+public class BusbarSectionPositionAdderImpl extends AbstractExtensionAdder<BusbarSection, BusbarSectionPosition>
+        implements BusbarSectionPositionAdder {
+
+    private int busbarIndex;
+
+    private int sectionIndex;
+
+    BusbarSectionPositionAdderImpl(BusbarSection busbarSection) {
+        super(busbarSection);
+    }
+
+    @Override
+    public BusbarSectionPositionAdder withBusbarIndex(int busbarIndex) {
+        this.busbarIndex = busbarIndex;
+        return this;
+    }
+
+    @Override
+    public BusbarSectionPositionAdder withSectionIndex(int sectionIndex) {
+        this.sectionIndex = sectionIndex;
+        return this;
+    }
+
+    @Override
+    protected BusbarSectionPosition createExtension(BusbarSection busbarSection) {
+        return new BusbarSectionPositionImpl(busbarSection, busbarIndex, sectionIndex);
```

Comment:
Maybe we could check that busbarIndex and sectionIndex have been defined and throw an exception otherwise.
```java
bbs.newExtension(BusbarSectionPositionAdder.class)..add();
```
=> create a (0, 0) extension but maybe this is not what is expected?